Split/Api/SplitClientConfig.swift

```diff
@@ -257,6 +257,6 @@ public class SplitClientConfig: NSObject {
 
     // This variable will be handled internaly based on
     // a random function
-    var isTelemetryEnabled = false
+    var isTelemetryEnabled = true
```

Comment:
Shouldn't this default to `false`?

Reply:
This is temporal y just for testing. That value will be populated by the random function specified in the spec.